src/main/java/com/cloudogu/wiki/RefreshWikisServlet.java

```diff
@@ -0,0 +1,33 @@
+package com.cloudogu.wiki;
+
+import com.cloudogu.wiki.scmm.SessionCacheScmWikiListStrategy;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+/**
+ * The RefreshWikisServlet removes cached wiki list from the session
+ *
+ * @author Maren Süwer
+ */
+public class RefreshWikisServlet extends HttpServlet {
+
+    private static final Logger LOG = LoggerFactory.getLogger(RefreshWikisServlet.class);
+
+    public RefreshWikisServlet() {}
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS1186)